lib/build.js

```diff
@@ -906,10 +906,10 @@ const buildAct = config => ({act, oog, pass, name, hasGas, gas, lemma}) => {
       && ("_ => " + buildReturns(act.returns))
       || act.returnsRaw && "_ => " + act.returnsRaw
       || ".WordStack"))
-    || "_ => _";
+    || "_ => ?_";
```

Comment:
@MrChico should this (and all other occurrences of `_ => _`) just be `_`?

Reply:
yeah

Reply:
I'm not convinced it should be "all" occurrences...there's a semantic difference between just `_` and `_ => ?_`. The former asserts that _no change occurs_, whereas the latter allows for a change but puts no constraint on what happens.

So things we really expect never to change during execution can be just `_`, everything else needs a rewrite (or so I believe based on empirical evidence).

Reply:
As for this specific occurrence...well, here's an alternative factoring of the logic (it's rather hard to read the current format):

```
  const output = (!pass || oog && "_ => ?_")
    || (act.internal   && "VOutput => VOutput")
    || (act.returns    && ("_ => " + buildReturns(act.returns)))
    || (act.returnsRaw && ("_ => " + act.returnsRaw           ))
    || ".WordStack";
```

So, for a `fail` or `oog` spec, do we expect `output` to never change? I think the answer is "yes", so in principle we can change to just `_`...but I'd also ask, are we gaining anything from that?